packages/synapse-react-client/src/components/QueryWrapper/TableRowSelectionState.ts

```diff
@@ -1,25 +1,24 @@
-import { ColumnModel, Row } from '@sage-bionetworks/synapse-types'
+import { Row, SelectColumn } from '@sage-bionetworks/synapse-types'
 import { isEqual } from 'lodash-es'
 import { atom } from 'jotai'
-import { tableQueryDataAtom } from './QueryWrapper'
 
 export function getRowSelectionEqualityComparator(
   row: Row,
-  columnModels?: ColumnModel[],
+  selectColumns?: SelectColumn[],
```

Comment:
For row selection, use the `SelectColumn`s. Since the `SelectColumn`s actually represent the table row data, it's more accurate. Also, the `SelectColumn`s are available in the `rowSet`, so we don't need to wait for the expensive facet/query count computations to happen first!